### PR TITLE
Enable Add to Wallet button for non-iOS devices with warning messages

### PR DESCRIPTION
**Pull request by samuelyeungkc**
As stated in #7, there are other apps besides iOS devices that support `.pkpass` file type. We want the app to be used as widely as possible. Therefore, it makes sense to allow other devices to download the file and add to their devices. 
A warning message is displayed when device is non-Apple as we cannot guarantee whether such device do support the file type. If user choose to download the file and unable to add to their apps it will be up to them to find a compatible apps.

PS. I am unable to build the project using the README instructions. If anyone can test this out it will be appricated.

### CODE REVIEW

File: components/Form.tsx [hunk 1]
```diff
@@ -328,6 +338,10 @@ function Form(): JSX.Element {
             setErrorMessage('Sorry, only Safari can be used to add a Wallet Pass on iOS');
             setIsDisabledAppleWallet(true);
             console.log('not safari')
+        } else if (!isIOS) {
+            setWarningMessage('Only Safari is officially supported at the moment. ' +
+                'Please download a compitable apps on other platform to open .pkpass');
```
Comment (straxus) [inline components/Form.tsx:343]: Typos - let's reword this as:
"Only Safari on iOS is officially supported for Wallet import at the moment - for other platforms, please ensure you have an application which can open Apple Wallet .pkpass files"

File: components/Form.tsx [hunk 2]
```diff
@@ -84,6 +85,15 @@ function Form(): JSX.Element {
         _setErrorMessages(Array.from(new Set([...errorMessages, translation !== message ? translation : message])));
     };
 
+    const setWarningMessage = (message: string) => {
+        if (message == undefined) {
```
Comment (straxus) [inline components/Form.tsx:89]: Can make this `if (!message) {` since we wouldn't want to display either null or '' either

And, I see that you copied this from setErrorMessage above - can you please update that function with this change as well?

**Review by straxus — COMMENTED** [review]
Thank you very much for submitting this PR! Once the noted changes have been made, I think we're good to merge!

File: components/Form.tsx (hunk 1 above)
Comment (samuelyeungkc) [reply components/Form.tsx]: this is updated. thx

File: components/Form.tsx (hunk 2 above)
Comment (samuelyeungkc) [reply components/Form.tsx]: updated both functions

**Review by straxus — APPROVED** [review]
Thanks!
